site/en/docs/privacy-sandbox/attribution-reporting/prevent-duplication/index.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'Prevent duplication in reports'
+subhead: >
+  Learn how to prevent duplicates in both event-level and aggregatable reports.
+description: >
+  Learn how to prevent duplicates in both event-level and aggregatable reports.
+date: 2023-03-06
+updated: 2023-07-21
+authors:
+  - maudn
+---
+
+
+## Ignore reports you've already received
+
+When the browser retries sending reports due to network unavailability, it may send the same report multiple times.
+To mitigate this:
+* Upon receiving a report, check its `report_id`.
+* Ignore the report if you've already received a report with that same `report_id`.
```

Comment:
These seem to be sequential? If so, they'd be in a numbered list.

Reply:
Ran inline. thanks!